src/System.Net.Http.WinHttpHandler/src/System.Net.Http.WinHttpHandler.msbuild

```diff
@@ -15,6 +15,7 @@
     <CompileItem Include="$(CommonPath)\Interop\Windows\winhttp\Interop.winhttp.cs" />
     <CompileItem Include="$(CommonPath)\Microsoft\Win32\SafeHandles\SafeHandleZeroOrMinusOneIsInvalid.cs" />
     <CompileItem Include="$(CommonPath)\System\StringExtensions.cs" />
+    <CompileItem Include="$(CommonPath)\System\Net\HttpKnownHeaderNames.cs" />
```

Comment:
>  So, we should balance that against all of these optimizations (which we don't really understand the true quantitative benefits of).

What is the DLL size before and after this change?

My proposal was to simply create HttpKnownHeaderNames.Cookie.cs and HttpKnownHeaderNames.cs as partial and import within each project only what is needed. The split will be made whenever it's necessary to reduce the DLL size.